doc/content/getting-started/platform-guide/_index.md

```diff
@@ -21,6 +21,10 @@ For example, if you are planning to use the {{% gnse %}}, you would need:
 1. {{% gnse %}} board
 2. {{% gnse %}} application
 
+## Getting a device
+
+{{% gnse %}} is currently in Beta phase. Join the early access program and become one of the first to access the node at [genericnode.com](https://www.genericnode.com/)
```

Comment:
```suggestion
{{% gnse %}} is currently in Beta phase. Join the early access program and become one of the first to access the node at [genericnode.com](https://www.genericnode.com/).
```

Reply:
I feel the `.com.` is a bit weird, I will merge this for now since it's already deployed and add the dot in another future proof-reading link improvement PR

Reply:
I agree, maybe "Generic Node webpage" instead